src/Domain/CoordinateMaps/CoordinateMap.tpp

```diff
@@ -63,6 +63,24 @@ bool CoordinateMap<SourceFrame, TargetFrame, Maps...>::is_identity() const
       maps_, std::make_index_sequence<sizeof...(Maps)>{});
 }
 
+template <typename SourceFrame, typename TargetFrame, typename... Maps>
+bool CoordinateMap<SourceFrame, TargetFrame,
+                   Maps...>::inv_jacobian_is_time_independent() const noexcept {
+  return std::is_same<
```

Comment:
This tmpl trick is comparatively difficult to understand on a first reading. Can this be simplified by using `flat_any` with appropriate inversions to make the logic work out? (like `not tmpl::flat_any_v<bool_constant<not blah>...>`)?

Reply:
I change the function to `...is_time_dependent`, and used `tmpl::flat_any_v`. I think this makes the logic inside easier and also easier to use.